ui/pages/smart-transactions/smart-transaction-status-page/smart-transactions-status-page.test.tsx

```diff
@@ -209,11 +225,14 @@ describe('SmartTransactionStatusPage', () => {
         CHAIN_IDS.MAINNET
       ][1];
     latestSmartTransaction.status = SmartTransactionStatuses.CANCELLED;
-    requestState.smartTransaction = latestSmartTransaction;
+    requestState.smartTransaction = latestSmartTransaction as SmartTransaction;
     requestState.isDapp = true;
     const store = configureMockStore(middleware)(mockStore);
     const { queryByText, container } = renderWithProvider(
-      <SmartTransactionStatusPage requestState={requestState} />,
+      <SmartTransactionStatusPage
+        requestState={requestState}
+        {...{ onCloseExtension, onViewActivity }}
```

Comment:
It's a bit out of scope for this PR, but do you think you could add 2 little additional tests here? One for checking that `onCloseExtension` was called when a user clicked on the `Close extension` button and another one for checking that `onViewActivity` was called after clicking on the `View activity` button.

Reply:
Done.